doc/source/api/triggers.rst

```diff
@@ -0,0 +1,12 @@
+.. _triggers:
+
+Triggers
```

Comment:
Something might not have been rendered properly for the triggers API doc:

```
Performs a trigger action every time_gap seconds. :type time_gap: int :param time_gap: Amount of seconds to wait before doing a trigger action
```

Reply:
Interesting, will take a look 🤔 .